src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -151,7 +156,13 @@ class OPENVINO_GENAI_EXPORTS WhisperPipeline {
      */
     WhisperDecodedResults generate(const RawSpeechInput& raw_speech_input,
                                    OptionalWhisperGenerationConfig generation_config = std::nullopt,
-                                   ChunkStreamerVariant streamer = std::monostate());
+                                   StreamerVariant streamer = std::monostate());
+    // todo: double check removal version
+    OPENVINO_DEPRECATED("ChunkStreamerBase is deprecated. "
+                        "Use StreamerBase instead. Support will be removed in 2026.1")
```

Comment:
we can use 2026.0 as next major version after API is deprecated

Reply:
Set to 2026.0